python/pyHyp.py

```diff
@@ -22,7 +22,7 @@
 import os
 import numpy
 from mpi4py import MPI
-from . import hyp, MExt
```

Comment:
This is the change I'm not sure about. I think it's okay since the imported module `hyp` is never accessed directly.

Reply:
Can you explain this a little more. 
What do you mean by "`hyp` is never accessed directly".
And why would it be a problem if it was

Reply:
The module `hyp` imported here is never used, the code does not make any reference to it at all. Instead, the compiled module is imported a second time [here](https://github.com/nwu63/pyhyp/blob/doc-update/python/pyHyp.py#L562) using `MExt`. And this is the `self.hyp` object which is used throughout the code. See [here](https://github.com/mdolab/adflow/blob/master/python/pyADflow.py#L113) for ADflow, where `libadflow` is only referenced once, via the call to `MExt`, and not imported at the beginning of the file.

The issue came up with `autodoc`, which is used to process docstrings automatically. To do so, it needs to be able to import whichever module we want to document, in this case `pyHyp`. That would however require access to the compiled module `hyp` which is not always available (we do not want to compile Fortran in order to get docstrings to compile). There are ways around it, and I've done that [here](https://github.com/nwu63/pyhyp/blob/doc-update/doc/conf.py#L34) for numpy and scipy, both of which may not be available when building docs. I'm not 100% sure this works with compiled objects instead of modules though, and in any case it is cleaner if we do not import things that are not needed.